content/en/docs/v3.6/op-guide/recovery.md

```diff
@@ -18,13 +18,86 @@ Recovering a cluster first needs a snapshot of the keyspace from an etcd member.
 $ ETCDCTL_API=3 etcdctl --endpoints $ENDPOINT snapshot save snapshot.db
 ```
 
+Note that taking the snapshot from the `member/snap/db` file might lose data that has not been written yet, but is included in the wal (write-ahead-log) folder.
+
+## Status of a snapshot
+
+To understand which revision and hash a given snapshot contains, you can use the `etcdutl snapshot status` command:
+
+```sh
+$ etcdutl snapshot status member/snap/db -wtable
++---------+----------+------------+------------+
+|  HASH   | REVISION | TOTAL KEYS | TOTAL SIZE |
++---------+----------+------------+------------+
+| 7ef846e |   485261 |      11642 |      94 MB |
++---------+----------+------------+------------+
+```
+
 ## Restoring a cluster
 
+### Revision Difference
+
+When you are restoring a cluster, existing clients may perceive the revision going back by many hundreds or thousands. This is due to the fact that a given snapshot only contains the data lineage up until the point of when it was taken, whereas the current state might already be further ahead.
+
+This is particularly a problem when running Kubernetes using etcd, where controllers and operators may use so called `informers` which act as local caches and get notified on updates using watches. Restoring to an older revision may not correctly refresh the caches, causing unpredictable and inconsistent behavior in the controllers.
+
+When restoring from a snapshot in the context of either: known consumers of the watch API, local cached copies of etcd data or when using Kubernetes in general - it is highly recommended to restore using "revision bumps" below. 
+
+### Restoring from database file
```

Comment:
The snapshot can be the db file or a snapshot created by `etcdctl snapshot save` command.

```suggestion
### Restoring from snapshot
```

Reply:
updated